### PR TITLE
Feature/issue219

### PR DESCRIPTION
**Pull request by dabooz**
See issue 219 for details of what was changed.

### CODE REVIEW

File: governance/governance.go [hunk 1]
```diff
@@ -780,23 +773,8 @@ func (w *GovernanceWorker) RecordReply(proposal *citizenscientist.Proposal, prot
...
-				envAdds[config.ENVVAR_PREFIX+"CONFIGURE_NONCE"] = proposal.AgreementId
-				envAdds[config.COMPAT_ENVVAR_PREFIX+"CONFIGURE_NONCE"] = proposal.AgreementId
-			} else {
-				envAdds[config.ENVVAR_PREFIX+"CONFIGURE_NONCE"] = workload.WorkloadPassword
-				envAdds[config.COMPAT_ENVVAR_PREFIX+"CONFIGURE_NONCE"] = workload.WorkloadPassword
-			}
 			envAdds[config.ENVVAR_PREFIX+"HASH"] = workload.WorkloadPassword
-			// For workload compatibility, the DEVICE_ID env var is passed with and without the prefix. We would like to drop
-			// the env var without prefix once all the workloads have ben updated.
-			envAdds["DEVICE_ID"] = w.deviceId
```
Comment (linggao) [inline governance/governance.go]: I am not sure if DEVICE_ID should be deleted or not. Need to confirm from @bmpotter .

File: governance/governance.go [hunk 2]
```diff
@@ -780,23 +773,8 @@ func (w *GovernanceWorker) RecordReply(proposal *citizenscientist.Proposal, prot
 				glog.Errorf("Error: %v", err)
 			}
 			envAdds[config.ENVVAR_PREFIX+"AGREEMENTID"] = proposal.AgreementId
-			envAdds[config.COMPAT_ENVVAR_PREFIX+"AGREEMENTID"] = proposal.AgreementId
-			envAdds[config.ENVVAR_PREFIX+"CONTRACT"] = w.Config.Edge.DVPrefix + proposal.AgreementId
-			envAdds[config.COMPAT_ENVVAR_PREFIX+"CONTRACT"] = w.Config.Edge.DVPrefix + proposal.AgreementId
```
Comment (linggao) [inline governance/governance.go]: Need to confirm from @bmpotter that HZN_CONTRACT can be removed.

File: governance/governance.go (hunk 1 above)
Comment (dabooz) [inline governance/governance.go]: Bruce and I talked, it can definitely be dropped.

File: governance/governance.go (hunk 2 above)
Comment (dabooz) [inline governance/governance.go]: Bruce and I talked about all the code in this section and he already confirmed that every change was correct.

**Pull request merged**